source/retryable-reads/tests/estimatedDocumentCount-serverErrors.yml

```diff
@@ -1,9 +1,11 @@
 runOn:
     -
         minServerVersion: "4.0"
+        maxServerVersion: "4.9.0"
         topology: ["single", "replicaset"]
     -
         minServerVersion: "4.1.7"
+        maxServerVersion: "4.9.0"
```

Comment:
Note: In contrast to my earlier comments, this is a justified case of using two runOn requirements since `minServerVersion` differs between topologies.